### DISCUSSION TITLE
Schema 4.0 validity,  INSPIRE validator and UML update

### DISCUSSION THREAD

**Discussion by hallinpihlatie** [INSPIRE-MIF/helpdesk#175] [General]
Hi,

A few things.

When looking for INSPIRE application schemas at the [INSPIRE Knowledge](https://knowledge-base.inspire.ec.europa.eu/tools/xml-schemas_en) base I couldn't find the links to the 5.0 versions of the application schemas. The links don't go to GitHub, but to the old INSPIRE web site, where only the 3.0 and 4.0 versions are available. For example for [Land Cover Vector ](https://inspire.ec.europa.eu/schemas/lcv/index.html).
 
You can find all schema versions (3.0, 4.0 and 5.0) on GitHub, for example for LCV here
https://github.com/INSPIRE-MIF/application-schemas/tree/main/schemas/lcv

Is this intentional or a mistake?

Presently the 4.0 versions of the schemas are used by the INSPRIE Validator. Will you be able to run tests for 4.0 schema versions too after the 5.0 schemas are in use? ([https://github.com/INSPIRE-MIF/helpdesk-validator/issues/1027](url))

Are the 4.0 versions depricated from 01/2024 onwards? On this web site, it says that old schemas are valid for 2 years after they are depricated. Is this the case? https://knowledge-base.inspire.ec.europa.eu/tools/xml-schemas_en

Will the INSPIRE UML model be updated with the 5.0 schema changes? Is there a time table for that?

Thanks in advance for your help,
Lena

**Comment by sMorrone** [comment]
Hi Lena,
the new schemas will be made available also in the[ official repository](https://inspire.ec.europa.eu/schemas/index.html) as soon as the metadata validations for M&R 2023 are finished, next week at the latest. After that, a new version of the validator will be released that will  use the new schema versions. There will be a presentation at the MIG-T meeting next week to explain and clarify all possible doubts.
The INSPIRE UML model will be updated with the 5.0 schema by the end of next week at the latest. However, the new UMLs are already present in the updated TGs.

**Comment by hallinpihlatie** [comment]
Sounds great, many thanks!

**Comment by fabiovinci** [comment]
Dear Community,

the updated schemas will be made available in the next few hours, so you may experience issues accessing them and in the operation of the INSPIRE Reference Validator. We will notify you when the process is complete.

The [release 2024.1](https://github.com/INSPIRE-MIF/uml-models/releases/tag/v2024.1) of the UML models was published.
All the updated artefacts (EA project, HTML view, feature catalogue and mapping tables) are directly accessible at https://inspire-mif.github.io/uml-models/

**Comment by fabiovinci** [comment]
Dear @hallinpihlatie 

> Are the 4.0 versions depricated from 01/2024 onwards? On this web site, it says that old schemas are valid for 2 years after they are depricated. Is this the case?

The answer to your question was provided during the [78th MIG-T meeting](https://wikis.ec.europa.eu/display/InspireMIG/78th+MIG-T+meeting+2024-05-17).

The results of the discussion are below for your convenience:

> JRC proposes that old schemas (v.4.0), which are no longer valid after the changes resulting from IR amendments, are still considered valid for one INSPIRE M&R cycle (2024), i.e. the new schemas will be required starting from M&R 2025. The MIG-T agrees to the proposal (see [poll results](https://wikis.ec.europa.eu/download/attachments/127305972/poll-results.pdf?version=3&modificationDate=1715947759639&api=v2)). Nevertheless, the use of the new schemas is already enforced in the Validator to stimulate their adoption.
